DOCS/license-inclusion-principles.md

```diff
@@ -7,42 +7,41 @@ Determining whether a candidate license should be included on the SPDX License L
 * The submitted license must not match another license already on the SPDX License List as per the SPDX matching guidelines.
 * All OSI-approved licenses will be included on the SPDX License List.
   * This is consistent with early collaboration between SPDX and OSI (2011) to ensure that all OSI-approved licenses were represented on the SPDX License List. Also at that time, OSI began using the SPDX license identifiers in their URLs and on the license pages.
-* Binary-only software licenses or licenses that do not provide for the availability of the source code will not be included on the SPDX License List.
+* Software licenses that apply only to executables and do not provide for the availability of the source code will not be included on the SPDX License List.
+* The license has identifiable and stable text; it is not in the midst of drafting. The license steward is committed to not modifying after addition to the list and to versioning new versions in the future.
```

Comment:
```suggestion
* The license has identifiable and stable text; it is not in the midst of drafting.
* The license steward, if any, is committed to not modifying after addition to the list and to versioning new versions in the future.
```

Historical licenses may not have active stewards, either because they're no longer maintaining the license, or because they're defunct, &c.

Reply:
that seems like a good addition, thanks!